### PR TITLE
Calendar default view

### PR DESCRIPTION
**Pull request by matthiasmoritz**
This PR allows frio User to set a default view for the calendar grid.

### CODE REVIEW

File: src/Model/Event.php [hunk 1]
```diff
@@ -412,10 +412,12 @@ public static function getItemArrayForImportedId(int $event_id, array $item = []
 	public static function getStrings(): array
 	{
 		// First day of the week (0 = Sunday).
-		$firstDay = DI::pConfig()->get(DI::userSession()->getLocalUserId(), 'system', 'first_day_of_week', 0);
+		$firstDay		= DI::pConfig()->get(DI::userSession()->getLocalUserId(), 'system', 'first_day_of_week', 0);
+		$calendar_defaultView = DI::pConfig()->get(DI::userSession()->getLocalUserId(), 'system', 'calendar_defaultView', 'month');
```
Comment (MrPetovan) [inline src/Model/Event.php]: Please move `'system', 'calendar_defaultView'` to `'calendar', 'default_view'`.

Please also move `'system', 'first_day_of_week'` to `'calendar', 'first_day_of_week'`. For backward compatibility reasons, you'll have to keep referring to the previous config value.
```suggestion
		$firstDay    = DI::pConfig()->get(DI::userSession()->getLocalUserId(), 'calendar', 'first_day_of_week') ??
		               DI::pConfig()->get(DI::userSession()->getLocalUserId(), 'system', 'first_day_of_week') ?? 0;
		$defaultView = DI::pConfig()->get(DI::userSession()->getLocalUserId(), 'system', 'calendar_defaultView', 'month');
```
But when you save the value, it should only set `'calendar', 'first_day_of_week'`.

File: src/Module/Settings/Display.php [hunk 2]
```diff
@@ -188,6 +189,14 @@ protected function content(array $request = []): string
 			6 => DI::l10n()->t("Saturday")
 		];
 
+		$calendar_defaultView = DI::pConfig()->get(DI::userSession()->getLocalUserId(), 'system', 'calendar_defaultView', 'month');
+		$calendarViews = [
+			'month'		=> DI::l10n()->t("month"),
+			'agendaWeek'=> DI::l10n()->t("week"),
+			'agendaDay'	=> DI::l10n()->t("day"),
+			'listMonth' => DI::l10n()->t("list")
```
Comment (MrPetovan) [inline src/Module/Settings/Display.php]: Please use spaces for assignment alignment. Additionally, please use single quotes unless double quotes are needed.

**Review by MrPetovan — CHANGES_REQUESTED** [review]

**Comment by MrPetovan** [comment]
I've taken over the branch so that it can go out this release.

File: database.sql [hunk 3]
```diff
@@ -1,6 +1,6 @@
 -- ------------------------------------------
 -- Friendica 2022.12-rc (Giant Rhubarb)
--- DB_UPDATE_VERSION 1501
+-- DB_UPDATE_VERSION 1502
```
Comment (nupplaphil) [inline database.sql:3]: why? :)

**Review by nupplaphil — CHANGES_REQUESTED** [review]

File: database.sql (hunk 3 above)
Comment (MrPetovan) [reply database.sql:3]: To make sure the `update_1502` method is executed. I don't know if a `update_1501` method would be executed if the current DB version already is 1501, so I didn't take any chances.

File: src/Module/Settings/Display.php [hunk 4]
```diff
@@ -199,6 +202,14 @@ protected function content(array $request = []): string
 			BBCode::PREVIEW_LARGE    => DI::l10n()->t('Large Image'),
 		];
 
+		$calendar_default_view = DI::pConfig()->get(DI::userSession()->getLocalUserId(), 'calendar', 'default_view', 'month');
+		$calendarViews = [
+			'month'		 => DI::l10n()->t('month'),
+			'agendaWeek' => DI::l10n()->t('week'),
+			'agendaDay'	 => DI::l10n()->t('day'),
+			'listMonth'  => DI::l10n()->t('list')
```
Comment (nupplaphil) [inline src/Module/Settings/Display.php]:
```suggestion
			'month'      => DI::l10n()->t('month'),
			'agendaWeek' => DI::l10n()->t('week'),
			'agendaDay'  => DI::l10n()->t('day'),
			'listMonth'  => DI::l10n()->t('list')
```

**Review by nupplaphil — CHANGES_REQUESTED** [review]

**Comment by matthiasmoritz** [comment]
Thank you @MrPetovan, today I wondered, what happend to my PR because I have some time to code but - surprise - its already done.

**Comment by MrPetovan** [comment]
Yeah, I wanted to snuck it in the 2022.12 release!

**Pull request merged**